src/Symfony/Component/String/Tests/AbstractAsciiTestCase.php

```diff
@@ -55,6 +55,13 @@ public static function provideBytesAt(): array
         ];
     }
 
+    public function testUnwrap()
+    {
+        $expected = ['hello', 'world'];
+        $actual = UnicodeString::unwrap([new UnicodeString('hello'), new UnicodeString('world')]);
```

Comment:
One more thing: the shouldn't hardcode `UnicodeString` instead, the name of the class should be variable.
See the test case just below for wrap(), it does it already.

Reply:
so can i fix it or this pr is not useful ?